test/Microsoft.Health.Dicom.Tests.Integration/Microsoft.Health.Dicom.Tests.Integration.csproj

```diff
@@ -28,6 +28,7 @@
     <PackageReference Include="xunit" Version="2.4.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3" />
     <PackageReference Include="Microsoft.SqlServer.DACFx" Version="150.5164.1" />
+    <PackageReference Include="System.Data.SqlClient" Version="4.8.2" PrivateAssets="All" />
```

Comment:
I had some trouble with this package too because fo-dicom includes it. Did another reference to this package sneak in?

Reply:
The test fails to  locate related assembly, so I add this.